mission/functions/systems/consumables/fn_consume.sqf

```diff
@@ -62,6 +62,7 @@ if (isClass _item_config) then
 			_food = linearConversion [0, _consume, _mag_size, 0, _food, true];
 
 			if (_water == 0 && _food == 0 && _cures == 0) then {
+				// @dijksterhuis b... but .. this script is player local?!?
 				{["EmptyFoodItem"] call para_c_fnc_show_notification} remoteExecCall ["call",player];
```

Comment:
Need to make sure there are no `call`s or `spawn`s remote executed by any clients to minimise what they can `remoteExec` -- so this needs to be changed to

```sqf
["EmptyFoodItem"] remoteExecCall ["para_c_fnc_show_notification", player];
```